src/monic.jl

```diff
@@ -0,0 +1,42 @@
+struct Monic{T,OPs<:AbstractQuasiMatrix{T},NL} <: OrthogonalPolynomial{T}
+    P::Normalized{T,OPs,NL}
+    # X::AbstractMatrix{T} # Should X be stored? Probably not. 
+    # X would be the Jacobi matrix of the normalised polynomials, not for the monic polynomials.
+end
+# Will need to figure out what this should be exactly. 
+# Consider this internal for now until it stabilises.
+
+Monic(P::AbstractQuasiMatrix) = Monic(Normalized(P))
+Monic(P::Monic) = P
+
+Normalized(P::Monic) = P.P
+
+axes(P::Monic) = axes(P.P)
+
+orthogonalityweight(P::Monic) = orthogonalityweight(P.P)
+
+_p0(::Monic{T}) where {T} = one(T)
+
+show(io::IO, P::Monic) = print(io, "Monic($(P.P.P))")
+show(io::IO, ::MIME"text/plain", P::Monic) = show(io, P)
+
+function getindex(P::Monic{T}, x::Number, n::Int)::T where {T}
```

Comment:
Actually is it easier to just overload `recurrencecoefficients`? That will call `forwardrecurrence!` automatically.